launchflow/gcp/firebase.py

```diff
@@ -0,0 +1,73 @@
+import dataclasses
+from typing import List, Optional
+
+from launchflow.gcp.resource import GCPResource
+from launchflow.models.enums import ResourceProduct
+from launchflow.models.flow_state import EnvironmentState
+from launchflow.node import Depends, Outputs
+from launchflow.resource import ResourceInputs
+
+
+@dataclasses.dataclass
+class FirebaseProjectOutputs(Outputs):
+    pass
+
+
+@dataclasses.dataclass
+class FirebaseProjectInputs(ResourceInputs):
+    pass
+
+
+class FirebaseProject(GCPResource[FirebaseProjectOutputs]):
```

Comment:
nit: add class level docs

Reply:
same here

Reply:
same here, gonna punt to tomorrow so I can land the readme changes